Makefile

```diff
@@ -107,6 +107,10 @@ ifeq ($(ENABLE_KUBE_API),true)
 	SERVICE_REPLICAS_COUNT=1
 	AUTH_TYPE=local
 endif
+ifdef BIP_BUTANE_CONFIG
+	BOOTSTRAP_INJECT_DIR=$(ROOT_DIR)/sno-bootstrap-manifests/
```

Comment:
is it "git ignored"?
if not, can we put it under a directory that is git ignored?

Reply:
Fixed